src/python/grpcio/tests/unit/beta/_implementations_test.py

```diff
@@ -48,6 +48,19 @@ def test_application_provided_root_certificates(self):
     self.assertIsInstance(
         channel_credentials, implementations.ChannelCredentials)
 
+class CallCredentialsTest(unittest.TestCase):
+
+  def test_google_call_credentials(self):
+    class MockGoogleCreds:
```

Comment:
Missing inheritance from `object` - but why are you using a mock here anyway? Why not use a real instance?

[I have some opinions on the subject](https://www.youtube.com/watch?v=Xu5EhKVZdV8).